src/lib.rs

```diff
@@ -200,6 +200,8 @@ pub use libp2p_wasm_ext as wasm_ext;
 pub use libp2p_websocket as websocket;
 #[doc(inline)]
 pub use libp2p_yamux as yamux;
+#[doc(inline)]
```

Comment:
Nit: I think the crates are sorted alphabetically here.

Reply:
Thanks @tomaka - will fix it.

Reply:
Fixed.